Jenkinsfile.hardware

```diff
@@ -0,0 +1,133 @@
+#!groovy
+
+pipeline {
+
+    agent {
+
+        node { 
+
+            label 'HardwareSensorBox' 
+
+        }
+
+    }
+
+    environment {
```

Comment:
Generally to the environment block: please resync it with the current upstream version of 'Jenkinsfile' to have the better structured block in new files as well.